hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -1,11 +1,72 @@
 module HookContextExtension
+  # FIXME: remove when #23332 is released
+  def param_value(mod, name)
+    param(mod, name).value if param(mod, name)
+  end
+
   def success_file
     File.join(File.dirname(kafo.config.config_file), '.installed')
   end
 
   def new_install?
     !File.exist?(success_file)
   end
+
+  def install_package(package)
```

Comment:
Bikeshedding: would `ensure_package_installed` be more better? When unfamiliar with it, you may want to still check if it's installed to optimize, even though it's not needed.